field_audit/models.py

```diff
@@ -321,7 +321,9 @@ def get_delta_from_instance(cls, instance, is_create, is_delete):
             "is_create and is_delete cannot both be true"
         fields_to_audit = cls.field_names(instance)
         # fetch (and reset for next db write operation) initial values
-        old_values = {} if is_create else cls.reset_initial_values(instance)
+        # NOTE: even if is_create is True, we need to call reset_initial_values
```

Comment:
Am I correct in interpreting this as "`reset_initial_values()` must be unconditionally called in order to prepare the instance for the next audit event"? If so, that might be something worthwhile clarifying in the docstring.

Reply:
Yeah agreed. See 59c266ad88866a2b21137676cf0b47ecebb223dd